### ISSUE TITLE
Simulate large sporting and cultural events

### ISSUE THREAD

**Issue by phelps-sg** [mrc-ide/covid-sim#210]
The going ahead of Cheltenham festival in the UK April 2020 was highly controversial.  Large "super-spreader" events such as this have the potential to spread the virus.  Calendars, numbers and venues for these events are available in the public domain.  It could be relevant to incorporate these events into the simulation model.

**Comment by robscovell-ts** [comment]
I agree that this is an important addition to the model. I have mentioned New Zealand before in another comment. A significant cluster in New Zealand centred on a farmers' convention.
